README

```diff
@@ -0,0 +1,11 @@
+# azure-translator-python-sdk
```

Comment:
so I have had issues with packaging the thing and putting it on pypi, so that's why. I'll revert as soon as I make it work properly